src/WebJobs.Extensions.DurableTask/LinuxAppServiceLogger.cs

```diff
@@ -100,18 +110,30 @@ private string GenerateJsonStr(EventWrittenEventArgs eventData)
             // We pack them into a JSON
             JObject json = new JObject
             {
-                { "EventStampName", this.stamp },
-                { "EventPrimaryStampName", this.primaryStamp },
                 { "ProviderName", eventData.EventSource.Name },
                 { "TaskName", eventData.EventName },
                 { "EventId", eventData.EventId },
                 { "TimeStamp", DateTime.UtcNow },
-                { "RoleInstance", this.roleInstance },
-                { "Tenant", this.tenant },
                 { "Pid", this.procID },
                 { "Tid", Thread.CurrentThread.ManagedThreadId },
             };
 
+            if (!string.IsNullOrEmpty(this.stamp) && !string.IsNullOrEmpty(this.primaryStamp))
```

Comment:
Is there a reason these environment variables were null?

Reply:
Well, in my test, the `tenant` (`"WEBSITE_STAMP_DEPLOYMENT_ID`) variable was empty.

As a result, and for safety, I made extra checks for all variables that I can't guarantee are set. That includes the stamp name (`"WEBSITE_HOME_STAMPNAME`") and, **when running in dedicated**, the container name.

Reply:
I suppose I could re-run the smoke test and see if we **actually** need to be careful with the stamp name, but I'm under the impression that these environment variables may change in the future and so I wanted to be extra careful.